File: CRS-Group-A-Java-Spring-REST/src/main/java/com/lti/bean/Student.java
package com.lti.bean;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

import org.springframework.stereotype.Component;

@Component
public class Student {
	
	private int id;
	private String name;
	private int majorId;
	private String email;
	private String phone;
	private String address;
	
	public Student() {}
	
	public Student(int id, String name, int majorId, String email, String phone, String address) {
		
		this.id = id;
		this.name = name;
		this.majorId = majorId;
		this.email = email;
		this.phone = phone;
		this.address = address;
	}
	
	

	/**
	 * @return the email
	 */
	public String getEmail() {
		return email;
	}

	/**
	 * @param email the email to set
	 */
	public void setEmail(String email) {
		this.email = email;
	}

	/**
	 * @return the phone
	 */
	public String getPhone() {
		return phone;
	}

	/**
	 * @param phone the phone to set
	 */
	public void setPhone(String phone) {
		this.phone = phone;
	}

	/**
	 * @return the address
	 */
	public String getAddress() {
		return address;
	}

	/**
	 * @param address the address to set
	 */
	public void setAddress(String address) {
		this.address = address;
	}

	/**
	 * @return the major
	 */
	public int getMajorId() {
		return majorId;
	}

	/**
	 * @param major the major to set
	 */
	public void setMajorId(int majorId) {
		this.majorId = majorId;
	}

	/**
	 * @return the id
	 */
	public int getId() {
		return id;
	}

	/**
	 * @param id the id to set
	 */
	public void setId(int id) {
		this.id = id;
	}

	/**
	 * @return the name
	 */
	public String getName() {
		return name;
	}

	/**
	 * @param name the name to set
	 */
	public void setName(String name) {
		this.name = name;
	}

}

File: CRS-GROUP-A-Angular-UI-UX-App/src/app/models/student.ts
export class Student {
	
	private id: number;
	private name: string;
	private majorId: number;
	private email: string;
	private phone: string;
	private address: string;
	
	constructor(id: number, name:string, majorId: number, email: string, phone: string, address: string) {
		
		this.id = id;
		this.name = name;
		this.majorId = majorId;
		this.email = email;
		this.phone = phone;
		this.address = address;
	}
	
	getEmail(): string {
		return this.email;
	}

	setEmail(email: string) {
		this.email = email;
	}

	getPhone(): string {
		return this.phone;
	}

	setPhone(phone: string) {
		this.phone = phone;
	}

	getAddress(): string {
		return this.address;
	}

	setAddress(address: string) {
		this.address = address;
	}

	getMajorId(): number {
		return this.majorId;
	}

	setMajorId(majorId: number) {
		this.majorId = majorId;
	}

	getId(): number {
		return this.id;
	}

	setId(id: number) {
		this.id = id;
	}

	getName(): string {
		return this.name;
	}

	setName(name: string) {
		this.name = name;
	}

}